tests/test_automation/jenkins_rhea_cpu.sh

```diff
@@ -43,7 +44,32 @@ fi
 make -j 24
 
 ctest -L unit
-#ctest -R short-LiH_dimer_ae-vmc_hf_noj-16-1
+
+
+echo "starting new test for real mixed precision"
+rm -rf ./*
```

Comment:
This typically dangerous. Can we create build_XXX folders to hold different builds?

Reply:
What are you worried about here?

Reply:
Note that the CI builds have never persisted, so creating new folders and then deleting them is the same as the current strategy.

Reply:
We do need to stay cognizant of disk issues, so that we don't run up against quotas, VM issues, etc. This will only get worse as time goes on and requirements expand. For troubleshooting, a human needs to manually build and troubleshoot the offending patches to post a fix for them anyway.
As for `rm -rf ./*`, it is indeed a dangerous tool that can burn the user. But similarly to lighting a fire, there are times when it is the best way to accomplish the goal. I don't think that keeping old CI builds around is a winning strategy, and so I need a way to clear out this directory. I'm open to other suggestions.

Reply:
What about adding a "cd .." followed by "rm -rf ./specific_directory".  This should greatly narrow the possible range of undesirable outcomes.